opencensus/metrics/transport.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2019, OpenCensus Authors
```

Comment:
Note that this `transport` serves a different purpose than `opencensus.common.transports`.

On master:
```
exporter.export ->  exporter.transport.export -> exporter.emit -> ... -> exporter.create_time_series
```
vs. on this branch:
```
background_thread -> exporter.export -> exporter.create_time_series
```

I'm open to a different name or structure here.

Reply:
This serves a similar purpose to https://github.com/census-instrumentation/opencensus-go/blob/master/metric/metricexport/reader.go#L62, what about `interval_metric_reader`?

Reply:
High level question about this flow since I'm a little behind how `stackdriver-metrics` works. What happens when `measure_to_view_map.export` is called now? I see it is deprecated is it a no-op somehow?

Reply:
Nothing! We should be able to remove this and many other supporting classes (e.g. `ViewData`) once stackdriver and prometheus are migrated.

Reply:
@songy23 it's long, but maybe worth it for the consistency between packages.